packages/perseus/src/widgets/python-program/python-program.tsx

```diff
@@ -39,7 +40,13 @@ class PythonProgram extends React.Component<Props> implements Widget {
     }
 
     render(): React.ReactNode {
-        const url = getUrlFromProgramID(this.props.programID);
+        let url = getUrlFromProgramID(this.props.programID);
```

Comment:
I wonder if we even want this logic built into perseus anymore, or if we'd make the client specify it inside of the `generateUrl` function.

Reply:
I'd agree.  It seems like this is very Khan Academy-specific (well, I'd argue that both `cs-program` and `python-program` are _very- KA-specific.  

I think for now I'd like to avoid making major changes to this and deal with URL generation once we have a built URL (instead of dealing with mapping of program ID to URL as part of this work).  But you make a good point!